docker/nginx/conf.d/http/include.visualization.template

```diff
@@ -0,0 +1,10 @@
+location /visualization {
+  include /etc/nginx/conf.d/include.closed_check;
+
+  proxy_pass http://$VISUALIZATION_HOST:$VISUALIZATION_HTTP_PORT;
+  proxy_set_header X-Real-IP $remote_addr;
+  proxy_set_header X-Scheme $scheme;
+  proxy_set_header Upgrade $http_upgrade;
+  proxy_http_version 1.1;
+  proxy_request_buffering off;
```

Comment:
`proxy_request_buffering` should be 'on' as it is by default until we use chunked streaming requests